ui_advanced_config.py

```diff
@@ -25,116 +28,82 @@ def __init__(self, master=None, logger=None):
         self.grid_columnconfigure(0, weight=1)
         self.grid_columnconfigure(1, weight=1)
         self.grid_columnconfigure(2, weight=1)
+        self.FixedGunKeys = [
+            "reload_ticks",
+            "reload_ticks_remaining",
+            "reloading",
+            "ammunition",
+            "min_damage",
+            "max_damage",
+            "range",
+        ]
+        self.EngineKeys = [
+            "min_speed",
+            "max_speed",
+            "cur_speed",
+            "max_turnrate",
+            "cur_turnrate",
+        ]
+        self.RadarKeys = [
+            "active",
+            "range",
+            "level",
+            "visarc",
+            "offset_angle",
+            "resolution",
+        ]
+        self.CnCKeys = ["max_cmds_per_tick"]
+        self.RadioKeys = ["max_range", "cur_range", "message"]
+        self.ArmKeys = ["max_weight", "max_bulk", "item"]
+
+
         self.BuildUI()
 
     def BuildUI(self):
 
-        # Make main containers
+        # Make main widgets
+
         self.mainFrame = uiQuietFrame(master=self)
         self.teamsColumn = uiQuietFrame(master=self.mainFrame)
         self.componentsColumn = uiQuietFrame(master=self.mainFrame)
         self.objectsColumn = uiQuietFrame(master=self.mainFrame)
         self.mapsColumn = uiQuietFrame(master=self.mainFrame)
-
-        # Make Widgets
         self.titleLabel = uiLabel(master=self.mainFrame, text="Advanced Settings")
 
+        # Place main widgets
+        self.mainFrame.pack(fill=tk.BOTH, expand=True, padx=10, pady=10)
+        self.titleLabel.pack(side=tk.TOP, fill=tk.BOTH, expand=False, padx=10, pady=10)
+
+        # Make Team Widgets
+
         self.selectTeamCombo = uiComboBox(master=self.teamsColumn)
         self.teamsLabel = uiLabel(master=self.teamsColumn, text="Teams")
         self.teamSizeLabel = uiLabel(master=self.teamsColumn, text="Size:")
```

Comment:
On testing, updating "t" size value to 10 has unintended consequence of changing TeamAlpha size to 1. Photo attached.
![image](https://github.com/Leading-AI-Development-Solutions/MAIA/assets/97774046/3100a550-d15a-433f-8eba-3b69ba03e3a1)

Reply:
For some reason I think this has to do with the fact that the teams.json has 2 TeamAlphas in it